apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -59,6 +61,10 @@ const pidIssuerClient = new PidIssuerClient(
   config.federationEntity.basePath.href,
 );
 
+appInsights.setup().start();
```

Comment:
if you want to be sure to send all ai data, you can use `setUseDiskRetryCaching()` at the expenses of some performance